src/index.js

```diff
@@ -149,6 +155,9 @@ type Props = {
   children?: any,
   style?: Object,
   childStyle?: Object,
+  element?: String,
```

Comment:
In flow, string literals are expressed as lowercase `string`. And this should IMO be `string | ReactClass`